lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -38,7 +38,8 @@ def search(query, language: nil, replacements: {}, subauth: nil, context: false)
         url = authority_service.build_url(action_config: search_config, action: :search, action_request: query, substitutions: replacements, subauthority: subauth, language: @language)
         Rails.logger.info "QA Linked Data search url: #{url}"
         load_graph(url: url)
-        parse_search_authority_response
+        results = process_results
+        convert_results_to_json(results)
```

Comment:
Refactor: Splits getting results from normalizing results following the principle of single responsibility.